scripts/prepare-badger-integration-tests.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import yaml
+import os
+
+config_file = os.path.join(os.path.dirname(__file__), '../cmd/jaeger/badger_config.yaml') 
+
+with open(config_file, 'r') as f:
+    config = yaml.safe_load(f)
+if 'badger_cleaner' not in config['service']['extensions'] :
+    config['service']['extensions'].insert(1, 'badger_cleaner') 
+
+config['extensions']['badger_cleaner'] = {
+    'trace_storage': 'badger_main'
+}
+
+with open(config_file, 'w') as f:
+    yaml.dump(config, f, sort_keys=False)
```

Comment:
You don't want to change the file in place, because then it can be accidentally committed. Let's create a new temporary file.

Reply:
modified the script to create a temp file for the new config. and also added a mechanism in the test to remove the temp config file.